### PR TITLE
t-puvasu/scan AntiVirus scan capabilities

### PR DESCRIPTION
**Pull request by ppvasude**
PR to install clamAV inside KuduLite container so that we can carry out the antivirus scans effectively.
Main tasks here are installing clamAV. Copying over the bash scrit which has the clamav run commands onto the container.

### CODE REVIEW

File: kudu/Dockerfile [hunk 1]
```diff
@@ -56,9 +56,9 @@ RUN cd /opt/dotnet && unlink lts && ln -s 2.2 /opt/dotnet/lts
 
 #Instal Kudu
 RUN cd /tmp \
-    && git clone https://github.com/Azure-App-Service/KuduLite.git \
+    && git clone https://github.com/ppvasude/KuduLite.git \
     && cd ./KuduLite \
-    && git checkout dev \
+    && git checkout user/t-puvasu/scan \
```
Comment (sanchitmehta) [inline kudu/Dockerfile]: Change this to `dev`

File: kudu/Dockerfile [hunk 2]
```diff
@@ -56,9 +56,9 @@ RUN cd /opt/dotnet && unlink lts && ln -s 2.2 /opt/dotnet/lts
 
 #Instal Kudu
 RUN cd /tmp \
-    && git clone https://github.com/Azure-App-Service/KuduLite.git \
+    && git clone https://github.com/ppvasude/KuduLite.git \
```
Comment (sanchitmehta) [inline kudu/Dockerfile]: Change it to the Azure-App-Service repo

File: kudu/Dockerfile [hunk 3]
```diff
@@ -76,6 +76,15 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 
 EXPOSE 8181
 
+# Install ClamAV
+RUN apt-get install -y clamav
```
Comment (sanchitmehta) [inline kudu/Dockerfile]: Move this part to the `# Install dependencies` part each run adds another layer. We want to minimize the number of Docker Image layers.

File: kudu/Dockerfile [hunk 4]
```diff
@@ -76,6 +76,15 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
...
+RUN freshclam
+
+# Copy custom scripts
+RUN mkdir /custom_scripts
+COPY custom_scripts/* /custom_scripts/
+RUN chmod 755 /custom_scripts/*
```
Comment (sanchitmehta) [inline kudu/Dockerfile:86]: Combine these run commands

File: kudu/Dockerfile [hunk 5]
```diff
@@ -76,6 +76,15 @@ ENV PATH=$PATH:/opt/nodejs/9/bin
 
 EXPOSE 8181
 
+# Install ClamAV
+RUN apt-get install -y clamav
+RUN freshclam
```
Comment (sanchitmehta) [inline kudu/Dockerfile]: Move this part to the `# Install dependencies` part

**Review by sanchitmehta — COMMENTED** [review]
Looks Good with minor changes!

File: kudu/Dockerfile (hunk 5 above)
Comment (ppvasude) [reply kudu/Dockerfile]: Done

File: kudu/Dockerfile (hunk 3 above)
Comment (ppvasude) [reply kudu/Dockerfile]: Done

File: kudu/Dockerfile (hunk 2 above)
Comment (ppvasude) [reply kudu/Dockerfile]: Done

File: kudu/Dockerfile (hunk 1 above)
Comment (ppvasude) [reply kudu/Dockerfile]: Done

File: kudu/Dockerfile (hunk 4 above)
Comment (ppvasude) [reply kudu/Dockerfile:86]: I ran a few manual tests on this, looks like we first need to copy the folder onto the container then run the chmod 755 for all the files inside that folder to get the permissions. Hence, combining the run command and then running the copy might not work in this case as the files inside /custom_scripts will not get the 755 permissions.

**Review by sanchitmehta — APPROVED** [review]

**Review by sanchitmehta — APPROVED** [review]

**Pull request merged**